primitives/tests/advz.rs

```diff
@@ -0,0 +1,41 @@
+#![cfg(feature = "test-srs")]
+use ark_bls12_381::Bls12_381;
+use ark_ff::{Field, PrimeField};
+use jf_primitives::{
+    pcs::{checked_fft_size, prelude::UnivariateKzgPCS, PolynomialCommitmentScheme},
+    vid::advz::Advz,
+};
+use sha2::Sha256;
+
+mod vid;
+
+#[test]
+fn round_trip() {
```

Comment:
a strange thing is this integration test is [not run inside CI](https://github.com/EspressoSystems/jellyfish/actions/runs/5449191040/jobs/9913183995#step:13:223). idk why.

Reply:
Hmm. My guess is it's because `tests/advz.rs` has `#![cfg(feature = "test-srs")]` on line 1. I did this because tests do not compile without it:
```
no function or associated item named `gen_srs_for_testing` found for struct `UnivariateKzgPCS` in the current scope
```
`gen_srs_for_testing` is feature-gated according to `#[cfg(any(test, feature = "test-srs"))]`. Seems like `test` is having no effect in this integration, so I need to gate behind `test-srs`. wat do 🤷 

Aside: yet another basic usability issue in Rust. I wish they'd fix this stuff already.

Reply:
Added `test-srs` feature flag to test script in https://github.com/EspressoSystems/jellyfish/pull/337/commits/639e51d0990f62ef3c2402715b23a241bfaa01cf . This test [now runs in CI](https://github.com/EspressoSystems/jellyfish/actions/runs/5456171771/jobs/9928625023?pr=337#step:13:370). 🎉